src/main/java/uk/gov/pay/adminusers/service/ServiceInviteCreator.java

```diff
@@ -93,7 +96,7 @@ public Invite doInvite(InviteServiceRequest inviteServiceRequest) {
                         return null;
                     });
                 })
-                .orElseThrow(() -> internalServerError(format("Role [%s] not a valid role for creating a invite service request", inviteServiceRequest.getRoleName())));
+                .orElseThrow(() -> internalServerError("Unable to retrieve admin service role"));
```

Comment:
This is how admin users tends to do things but explicitly throwing something called an "internalServerError" feels wrong and not very helpful from an error stack/ code readability perspective

As this ends up as a 500 with a generic message anyway I wonder if it would be OK to throw a named exception that just gets picked up by the default dropwizard error mapper

```suggestion
                .orElseThrow(() -> new AdminRoleNotFoundInDBYikes());
```

Reply:
I would argue that named exceptions are only useful when we need to handle them in a special way or have specific exception mapper for them so I'd prefer not to add the unnecessary boilerplate.

Reply:
This specific exception will disappear very soon when we stop creating a service as part of a self-sign up

Reply:
Yeah I think all good especially as it's unlikely to be used elsewhere